src/frontend/apps/impress/src/features/docs/doc-header/components/blocks/Table.tsx

```diff
@@ -0,0 +1,60 @@
+import { TD, TH, TR, Table as TablePDF } from '@ag-media/react-pdf-table';
+import {
+  DefaultBlockSchema,
+  Exporter,
+  InlineContentSchema,
+  StyleSchema,
+  TableContent,
+} from '@blocknote/core';
+import { View } from '@react-pdf/renderer';
+import { ReactNode } from 'react';
+
+export const Table = (props: {
+  data: TableContent<InlineContentSchema>;
+  transformer: Exporter<
+    DefaultBlockSchema,
+    InlineContentSchema,
+    StyleSchema,
+    unknown,
+    unknown,
+    unknown,
+    unknown
+  >;
+}) => {
+  return (
+    <TablePDF>
+      {props.data.rows.map((row, index) => {
+        if (index === 0) {
+          return (
+            <TH key={index}>
+              {row.cells.map((cell, index) => {
+                return (
+                  <TD key={index}>
```

Comment:
I have a few warning about identical key, maybe it can be fix easily ?

![image](https://github.com/user-attachments/assets/1f091a1d-de49-41cd-92c8-f1b17c422244)

Reply:
I do not have these errors, can you give me your doc so I can try on my side?